zipline/finance/performance/tracker.py

```diff
@@ -314,18 +333,17 @@ def process_dividend(self, dividend):
     def process_split(self, event):
         leftover_cash = self.position_tracker.handle_split(event)
         if leftover_cash > 0:
-            for perf_period in self.perf_periods:
-                perf_period.handle_cash_payment(leftover_cash)
+            self.cumulative_performance.handle_cash_payment(leftover_cash)
+            self.todays_performance.handle_cash_payment(leftover_cash)
 
     def process_order(self, event):
-        for perf_period in self.perf_periods:
-            perf_period.record_order(event)
+        self.cumulative_performance.record_order(event)
```

Comment:
can we rename these parameters to something more specific , like "order" in this case?  and if you don't mind, can you put some docstrings on these methods?